tools/mo/openvino/tools/mo/back/ConvolutionNormalizer.py

```diff
@@ -29,22 +33,69 @@ def resolve_convolution_with_group(node: Node, group: int, ir_version: str):
         if weights_shape[0] == node.output:
             # weights are already is in [G*O I X Y] format
             return
-        new_shape = shape_array([node.output, -1, *weights_shape[2:]])
+
+        num_spatial_dims = len(weights_shape[2:])
+        # Reshape has special_zero=True, if zeros are set then original shapes are copied
+        zeros_to_copy_spatial_dims = np.zeros(num_spatial_dims)
+        new_shape = shape_array([node.output, -1, *zeros_to_copy_spatial_dims])
+        reshape = create_op_node_with_second_input(node.graph, Reshape, new_shape,
+                                                   {'override_output_shape': True})
     elif ir_version == 'V10':
-        # TODO rewrite this transformation to generate a shape-computing sub-graph. Ticket 62076
         I = input_shape[1]
-        new_shape = shape_array([group, node.output // group, I // group, *weights_shape[2:]])
-        assert is_fully_defined(weights_shape[2:]) and is_fully_defined(I) and \
-               np.prod(weights_shape) == np.prod(new_shape), 'Initial weights shape {}, grouped weights shape {}' \
-                                                             ''.format(weights_shape, new_shape)
+        if is_fully_defined(weights_shape[2:]) and is_fully_defined(I):
+            new_shape = shape_array([group, node.output // group, I // group, *weights_shape[2:]])
+            assert np.prod(weights_shape) == np.prod(new_shape), 'Initial weights shape {}, grouped weights shape {}' \
+                                                                 ''.format(weights_shape, new_shape)
+            reshape = create_op_node_with_second_input(node.graph, Reshape, new_shape,
+                                                       {'override_output_shape': True})
```

Comment:
please set special_zero that is mandatory attribute by the spec

Reply:
done